src/building/how-to-build-and-run.md

```diff
@@ -30,23 +30,57 @@ settings (and possibly others, such as `llvm.ccache`):
 
 ```toml
 [llvm]
-# Enables LLVM assertions, which will check that the LLVM bitcode generated
-# by the compiler is internally consistent. These are particularly helpful
-# if you edit `codegen`.
+# Indicates whether the LLVM assertions are enabled or not
 assertions = true
```

Comment:
Why do we recommend this? I don't expect most new contributors to be editing codegen and it will slow down the compiler.

Reply:
:+1: I think you're right here, I've just left it there because it was there already but given that we are fixing suggestions it may be a good idea to remove it.

What others think about this? why was this originally added?.

Reply:
Discussed here https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-rustc-dev-guide/topic/meeting.2007.2E14.2E2020